include/CRADLE/Utilities.hh

```diff
@@ -158,31 +237,59 @@ namespace utilities {
     return result;
   }
 
-  inline double L0Correction(double W, int Z, double r, int betaType) {
+  inline double L0Correction(double W, int Z, double r, int betaType) {     /////////// L0 Correction ---- L0 (voir specific, ajouter le sum)
                                        //double aPos[], double aNeg[]) {
     double gamma = std::sqrt(1. - std::pow(FINESTRUCTURE * Z, 2.));
     double sum = 0;
     double common = 0;
     double specific = 0;
-    /*for (int i = 1; i < 7; i++) {
-      if (betaType == BETA_PLUS)
+
+    /////////////////////// Completion du code SL 10/05/2023
+    double bNeg = 0;
+    double aNeg[1] = {0};
+
+    double bPos[7][6] = { {0.0701, -2.572, 27.5971, -128.658, 272.264, -214.925},
```

Comment:
Here, it would be better to use the full implementation as, for example, in the BSG code so that it is viable for all decays. If you don't have time to do this, I can do it afterwards.

Reply:
I didn't have to do this during my internship, I will fix this.